packages/react-query/src/scripts/utils/package.ts

```diff
@@ -1,51 +1,85 @@
+import fs from 'fs'
+import path from 'path'
 import { exit } from 'process'
-import { loadModule } from './loadModule'
+import { type LoadModuleResult, loadModule } from './loadModule'
 
 type PackageJson = {
   name: string
   description: string
   version: string
 }
 
-export function getPackageJson() {
-  const packageJson = loadModule<PackageJson>('@suspensive/react-query/package.json')
-  if (!packageJson) {
+export function getPackageJson(): PackageJson {
+  const module = loadModule<PackageJson>('@suspensive/react-query/package.json')
+  if (!module.isSuccess) {
     console.warn('@suspensive/react-query `package.json` is not found.')
     exit(1)
   }
 
-  return packageJson
+  return module.exports
 }
 
-export function getTanStackReactQueryPackageJson() {
-  const packageJson = loadModule<PackageJson>('@tanstack/react-query/package.json')
-  if (!packageJson) {
+export function getTanStackReactQueryPackageJson(): PackageJson {
+  const module = loadModule<PackageJson>('@tanstack/react-query/package.json')
+  if (!module.isSuccess) {
     console.warn('@tanstack/react-query is not found. Please install @tanstack/react-query.')
     exit(1)
   }
 
-  return packageJson
+  return module.exports
 }
 
-export function getSuspensiveReactQueryPackageJson(targetVersion: string) {
-  let packageJson: PackageJson | undefined
+export function getSuspensiveReactQueryPackageJson(targetVersion: string): PackageJson {
+  let module: LoadModuleResult<PackageJson>
 
   switch (targetVersion) {
     case '4':
-      packageJson = loadModule<PackageJson>('@suspensive/react-query-4/package.json')
+      module = loadModule<PackageJson>('@suspensive/react-query-4/package.json')
       break
     case '5':
-      packageJson = loadModule<PackageJson>('@suspensive/react-query-5/package.json')
+      module = loadModule<PackageJson>('@suspensive/react-query-5/package.json')
       break
     default:
       console.warn(`@suspensive/react-query-${targetVersion} is not found.`)
       exit(1)
   }
 
-  if (!packageJson) {
+  if (!module.isSuccess) {
     console.warn(`@suspensive/react-query-${targetVersion} is not found.`)
     exit(1)
   }
 
-  return packageJson
+  return module.exports
+}
+
+export function getIndexFileContent(...paths: string[]): string {
+  const basePath = path.resolve(...paths, 'dist').replace(/src/, '')
+
+  return fs.readFileSync(path.join(basePath, 'index.js'), 'utf-8') || ''
+}
+
+export function getTargetSuspensiveReactQueryVersion(): string {
+  const indexFileContent = getIndexFileContent(__dirname, '../../')
+
+  return (RegExp(/@suspensive\/react-query-(\d+)/).exec(indexFileContent) || [])[1] || ''
+}
+
+export function getTargetSuspensiveReactQueryAPIs(): string[] {
+  const indexFileContent = getIndexFileContent(__dirname, '../../')
+
+  const modules = indexFileContent.matchAll(/export \* from ['"](@suspensive\/react-query-\d+)['"]/g)
```

Comment:
I have changed it to extract APIs only from suspensive/react-query. This is to prevent the re-export of TanStack Query v5's APIs in the future. (I believe the re-export of TanStack Query should not be displayed in the status table.)

Reply:
In my opinion, re-exported apis of @suspensive/react-query should be displayed in status table. because actually they are exported by @suspensive/react-query too anyway.

I want to check why you believe the re-export of TanStack Query should not be displayed in the status table.

Reply:
I think the re-export of TanStack Query is not a feature provided by Suspensive and could cause confusion for users.

- The re-export of TanStack Query is a tool (hint) to help with migration from v4 to v5. (It cannot be used to leverage Suspensive.)
- If displayed in the Status Table, users might mistakenly use it as a Suspensive feature.

Reply:
> I think the re-export of TanStack Query is not a feature provided by Suspensive and could cause confusion for users.

In my opinion, Re-exporting of TanStack Query could be "feature" anyway provided by Suspensive's intent too.

> - If displayed in the Status Table, users might mistakenly use it as a Suspensive feature.

In my opinion, using re-exported apis is not mistake. because anyway we exports them as public apis. then, how about mark it as "re-exporting"

## Results expected

### srq status with @tanstack/react-query@4

|@suspensive/react-query|result|
|------|---|
|Available interfaces from @suspensive/react-query-4|useSuspenseQuery, useSuspenseQueries, useSuspenseInfiniteQuery, queryOptions, SuspenseQuery, SuspenseQueries, SuspenseInfiniteQuery, Mutation|
|Available interfaces from @tanstack/react-query@4||

### srq status with @tanstack/react-query@5

|@suspensive/react-query|result|
|------|---|
|Available interfaces from @suspensive/react-query-5|SuspenseQuery, SuspenseQueries, SuspenseInfiniteQuery, Mutation|
|Available interfaces from @tanstack/react-query@5|useSuspenseQuery, useSuspenseQueries, useSuspenseInfiniteQuery, queryOptions|

Reply:
Cool! I will implement it using that approach!

Reply:
@manudeli Perhaps this approach might make it more complicated for users. As you mentioned, re-export can be considered a "feature" (a feature that helps with migration). I would like to follow your opinion on how to display these APIs. What do you suggest we should do?

<img width="962" alt="image" src="https://github.com/user-attachments/assets/80113664-65b6-46e3-8845-d72258a706da">

Reply:
How about this?

|@suspensive/react-query|result|status|description|
|------|---|---|---|
|version|2.9.0|🟢||
|export|@suspensive/react-query-5@2.9.0|🟢|You can use SuspenseQuery, SuspenseQueries, SuspenseInfiniteQuery, Mutation|
|peerDependency|@tanstack/react-query@5.29.1|🟢|You can use useSuspenseQuery, useSuspenseQueries, useSuspenseInfiniteQuery, queryOptions|

|@suspensive/react-query|result|status|description|
|------|---|---|---|
|version|2.9.0|🟢||
|export|@suspensive/react-query-4@2.9.0|🔴|You should `npx srq switch 5` to fix this error|
|peerDependency|@tanstack/react-query@5.29.1|🟢|You can use useSuspenseQuery, useSuspenseQueries, useSuspenseInfiniteQuery, queryOptions|